<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAcTime</class>
 <widget class="QMainWindow" name="QAcTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAcTime</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="todayLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Сегодня: 01.08.2019</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="newActivityButton">
            <property name="text">
             <string>Новое</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseButton">
            <property name="text">
             <string>Приостановить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="continueButton">
            <property name="text">
             <string>Продолжить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/new-act.png</normaloff>:/images/new-act.png</iconset>
   </property>
   <property name="text">
    <string>newAction</string>
   </property>
   <property name="toolTip">
    <string>Новое</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>pauseAction</string>
   </property>
   <property name="toolTip">
    <string>Приостановить</string>
   </property>
  </action>
  <action name="continueAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/continue.png</normaloff>:/images/continue.png</iconset>
   </property>
   <property name="text">
    <string>continueAction</string>
   </property>
   <property name="toolTip">
    <string>Продолжить</string>
   </property>
  </action>
  <action name="editAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>editAction</string>
   </property>
   <property name="toolTip">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>deleteAction</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="reportAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
   </property>
   <property name="text">
    <string>reportAction</string>
   </property>
   <property name="toolTip">
    <string>Отчёт</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>aboutAction</string>
   </property>
   <property name="toolTip">
    <string>О программе</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset resource="qactime.qrc">
     <normaloff>:/images/qt-logo.png</normaloff>:/images/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>aboutQtAction</string>
   </property>
   <property name="toolTip">
    <string>Информация о Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qactime.qrc"/>
 </resources>
 <connections/>
</ui>
